model_access_restriction/tests/test_model_access_restriction.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import AccessError, ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestModelAccessRestriction(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.model = cls.env.ref("base.model_res_partner")
+        cls.model2 = cls.env.ref("base.model_res_partner_category")
```

Comment:
Do you need model2?

Reply:
I have changed the tests to use this model